src/json/json.cr

```diff
@@ -1,3 +1,84 @@
+# JavaScript Object Notation (JSON)
+#
+# JSON (JavaScript Object Notation) is a way to write data in Javascript.
+# Like XML, it allows to encode structured data in a text format that can
+# be easily read by humans Its simple syntax and native compatibility with
+# JavaScript have made it a widely used format.
+#
+# An object is a series of string keys mapping to values, in `"key": value`
+# format. Arrays are enclosed in square brackets `[ ... ]` and objects in
+# curly brackets `{ ... }`.
+#
+# Crystal provides a mechanism for encoding & decoding of values to and
+# from JSON via the serialization API.
+#
+# To read more about JSON visit: json.org
+#
+# Parsing JSON
+#
+# To parse a JSON string received by another application or generated within
```

Comment:
tbh i'm not sure the origin of the JSON is relevant. "To parse a JSON string, you can do the following" or just "To parse a JSON string:"